src/topology/config/mod.rs

```diff
@@ -39,6 +40,8 @@ pub struct GlobalOptions {
     pub data_dir: Option<PathBuf>,
     #[serde(default)]
     pub dns_servers: Vec<String>,
+    #[serde(default)]
```

Comment:
is the default `false` here?

Reply:
It is. https://doc.rust-lang.org/src/core/default.rs.html#127-131